activerecord/lib/active_record/connection_adapters/sqlite3/database_statements.rb

```diff
@@ -128,20 +128,17 @@ def execute_batch(statements, name = nil)
             raw_execute(sql, name, batch: true)
           end
 
-          def build_fixture_statements(fixture_set)
-            fixture_set.flat_map do |table_name, fixtures|
-              next if fixtures.empty?
-              fixtures.map { |fixture| build_fixture_sql([fixture], table_name) }
-            end.compact
-          end
-
           def build_truncate_statement(table_name)
             "DELETE FROM #{quote_table_name(table_name)}"
           end
 
           def returning_column_values(result)
             result.rows.first
           end
+
+          def default_insert_value(column)
+            column.default
+          end
```

Comment:
SQLite doesn't support the `DEFAULT` keyword on inserts, so I'm using the default value from the schema.